modules/service/src/main/scala/lucuma/odb/service/CalibrationsService.scala

```diff
@@ -369,6 +419,17 @@ object CalibrationsService {
             WHERE t_program.c_calibration_role=$calibration_role
               AND t_program.c_existence='present'
           """.query(target_id *: right_ascension *: declination *: epoch *: int8.opt *: int8.opt *: radial_velocity.opt *: parallax.opt)
+
+    val selectCalibrationTimeAndConf: Query[Observation.Id, (Observation.Id, Option[Timestamp], Option[ObservingModeType])] =
+      sql"""
+        SELECT
+            c_observation_id,
+            c_observation_time,
+            c_observing_mode_type
+          FROM v_generator_params
+          WHERE c_observation_id = $observation_id AND c_calibration_role IS NOT NULL
+          """.query(observation_id *: core_timestamp.opt *: observing_mode_type.opt)
```

Comment:
I'm wondering about adding observation time to the generator params view.  It isn't really needed to generate the sequence, at least not yet.  All these things come from the observation itself in `t_observation`.  Is there any reason to not just query `t_observation` directly?